lib/command/src/Obelisk/Command/Utils.hs

```diff
@@ -130,6 +134,21 @@ gitLsRemote repository mRef = do
   putLog Debug $ "git ls-remote maps: " <> T.pack (show maps)
   pure maps
 
+gitLsRemoteExitCode
+  :: MonadObelisk m
+  => String
+  -> String
+  -> m (ExitCode, GitLsRemoteMaps)
+gitLsRemoteExitCode repository branchName = do
```

Comment:
This looks pretty similar to `gitLsRemote` maybe we should just combine the two or write one using the other